tests/test_savings_kpi.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
+from uuid import uuid4
+
+from d3a_interface.sim_results.kpi import SavingsKPI
+
+
+class TestSavingsKPI(TestCase):
+
+    def setUp(self) -> None:
+        self.savings_kpi = SavingsKPI()
+
+    def tearDown(self) -> None:
+        pass
+
+    def test_correct_classification_of_area_type(self):
```

Comment:
I would suggest naming unit tests including the name of the method you are testing. For example:
```suggestion
    def test_populate_area_type(self):
```
or
```suggestion
    def test_populate_area_type_is_correct(self):
```
So the format would be something like **<method_name>_<state_tested>_<expected_outcome>** or similar.